app/lib/theme/widgets/AdjustableCalculatedValueWidget.tsx

```diff
@@ -0,0 +1,85 @@
+import { WidgetProps } from "@rjsf/core";
+import NumberFormat from "react-number-format";
+
+/*
+  to get the calculated value from formContext, we need to tweak the label of the field to match the calculated value in the formContext.
+  for example, if the label is "Total Funding Request", we need to change it to "totalFundingRequest" and then add `CalculatedValue` to the end of the label.
+*/
+export const getFormattedLabel = (fieldLabel: string) =>
+  fieldLabel.charAt(0).toLowerCase() + fieldLabel.slice(1).replace(/\s/g, "");
+
+export const AdjustableCalculatedValueWidget: React.FC<WidgetProps> = (
+  props
+) => {
+  const {
+    schema,
+    uiSchema,
+    id,
+    disabled,
+    label,
+    onChange,
+    value,
+    formContext,
+  } = props;
+
+  // If we are using this widget to show numbers as money, we can set `isMoney` to true in the uiSchema.
+  const isMoney = uiSchema?.isMoney;
+
+  const calculatedValue =
+    formContext[`${getFormattedLabel(label)}CalculatedValue`];
+
+  return (
+    <div>
+      {calculatedValue && (
+        <>
+          <div style={{ marginBottom: "1em" }}>
+            {isMoney ? "$" : ""}
+            {calculatedValue}
+          </div>
+          <label htmlFor={id}>{label} (Adjusted)</label>
```

Comment:
By default, the `FieldTemplate` component already creates a label with `htmlFor={id}`, so this would be redundant. 
What I'd suggest here is to add both `<label>` fields for each of the calculated and adjusted values in here, and the developer can turn the `FieldTemplate` rendering on or off in the uiSchema with the `ui:options` values